bin/buildsite.py

```diff
@@ -110,7 +136,7 @@ def start_build(args):
     buildcmd = ('/bin/bash', '-c',
                 'source bin/activate; cd source && '
                 ### note: adding --debug can be handy
-                f'(pelican content --settings {settings_path} -o {buildpath})',
+                f'(pelican {content_dir} --settings {settings_path} -o {buildpath})',
```

Comment:
Strictly speaking `content` is relative to cwd, while `{content_dir}` is a longer constructed path. By tracing back the `os.path.join()` calls, it is seen that `content_dir` is an abspath. It *may* not be the same depending upon cwd (and I have no way to test within production), but I'm pretty confident that the cwd is correct for the original cwd-dependent `content` to succeed.

Reply:
oh. `cd source` two lines above.